game-core/src/main/java/tools/util/ToolLogger.java

```diff
@@ -2,19 +2,24 @@
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
+import java.util.logging.Level;
+
+import lombok.extern.java.Log;
+
 /**
  * Provides methods for support tools to write log messages.
  */
-public final class ToolLogger {
-  private ToolLogger() {}
+@Log
+public enum ToolLogger {
+  ;
```

Comment:
An enum with zero elements just to hold a few static methods smells a bit.  Is this an IDE recommended change?  Would be interested in understanding the reasoning.

**ALSO:** For a follow-up PR, this type should be completely removed and the `Logger` calls inlined.  The purpose of this type was to simply provide a counterpart to `ClientLogger` so devs wouldn't use `ClientLogger` within the map making tools (used to cause problems when the tools could run out-of-process).

Reply:
The enum replacement is similar to singleton with enum. The enum here is the same as a final class with private constructor, kinda the same transformation as with a singleton. The fact this is a utility method only class is a smell, I would agree.

Reply:
Got motivated and killed off this class: https://github.com/triplea-game/triplea/pull/3715/commits/23b017352130c1792edfd551f87834b8277171c6, this *is* to unify logging :grin: